### DISCUSSION TITLE
C++ VsCode intellisense over remote SSH doesn't work

### DISCUSSION THREAD

**Discussion by gmolkho** [microsoft/vscode-cpptools#7158] [Q&A]
I'm using  Win10 C/C++ for Visual Studio Code with Remote - SSH to work from over Ubuntu 18.04 station, and I can't get any intellisense features working - is it my own station problem, should I set something anywhere?

**Comment by sean-mcmanus** [comment]
Does the remote side have the C/C++ extension installed? Do you see any logging or error messages?
